lib/ship.rb

```diff
@@ -1,5 +1,5 @@
 class Ship
-  attr_reader :name, :length, :health
+  attr_reader :name, :length, :health, :hit
```

Comment:
(As we discussed in person), you added this to try to resolve a NoMethodError, but my suspicion is that--because there is already a .hit method, the problem wasn't that it needed a getter method but that you may have been invoking .hit on a ship that didn't exist (because it didn't get placed on that cell). So I think we can remove .hit from attr_reader.